app/src/main/java/com/neilkrishna/basicapplication/AddPeopleInfoActivity.kt

```diff
@@ -0,0 +1,55 @@
+package com.neilkrishna.basicapplication
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.Toast
+import androidx.lifecycle.Observer
+import androidx.room.Room
+import com.neilkrishna.basicapplication.databinding.ActivityAddPeopleInfoBinding
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+
+class AddPeopleInfoActivity : AppCompatActivity() {
```

Comment:
Plz implement the view model for insertion into the database and make use of View model coroutine scope instead of global scope. Also include a circular progress bar which will be displayed while the entry is inserted into the database.

Reply:
could you elaborate about circular progress bar? i didn't get it.

Reply:
U display the progress bar before u start the transaction to database and hide it as u complete the transaction